template.distillery/Makefile.os

```diff
@@ -82,8 +82,10 @@ DIST_FILES = $(ELIOMSTATICDIR)/$(PROJECT_NAME).js $(LIBDIR)/$(PROJECT_NAME).cma
 
 .PHONY: test.byte test.opt
 test.byte: $(TEST_CONFIG_FILES) $(addprefix $(TEST_PREFIX),$(DIST_DIRS) $(DIST_FILES)) css
+	@echo "==== The website is served port $(TEST_PORT) (http://localhost:$(TEST_PORT)) ===="
```

Comment:
"served port" sounds wrong. What about something simpler like "The website is available at `http://localhost:$(TEST_PORT)`" ?